app/services/claims/calculate_total_mentor_training_hours_for_provider.rb

```diff
@@ -0,0 +1,16 @@
+class Claims::CalculateTotalMentorTrainingHoursForProvider
+  include ServicePattern
+
+  def initialize(mentor:, provider:)
+    @mentor = mentor
+    @provider = provider
+  end
+
+  def call
+    mentor.mentor_trainings.where(provider:).sum(:hours_completed)
```

Comment:
I wonder if this service should also calculate hours per academic year. But for now this is enough I think

Reply:
Yeah, it _should_. But we'll ignore the concept of academic years for now. We'll be addressing academic years after the private beta.